components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.spec.ts

```diff
@@ -68,6 +68,18 @@ describe('ReportingComponent', () => {
     expect(qe).not.toBeNull();
   });
 
+  it('convertMomentToDateWithoutTimezone', () => {
```

Comment:
For consistency should be a verb-phrase rather than a noun, perhaps for this test:
```suggestion
  it('maps hour correctly without timezone', () => {
```
and for the next test, looks like _with_ timezone...?